src/main/java/com/nametagedit/plugin/packets/PacketWrapper.java

```diff
@@ -94,14 +94,14 @@ public PacketWrapper(String name, String prefix, String suffix, int param, Colle
                     PacketAccessor.PACK_OPTION.set(packet, 1);
 
                     if (PacketAccessor.VISIBILITY != null) {
-                        PacketAccessor.VISIBILITY.set(packet, visible ? "always" : "never");
+                        PacketAccessor.VISIBILITY.set(packetParams, visible ? "always" : "never");
```

Comment:
Not again 😅 
Pre-1.17, ``packet``, post-1.17 ``packetParams``, sorry, wasn't clear.

Reply:
So have I to put a check?

Reply:
No, I already implemented the check, loot at ``// 1.17+ comment``. Just like that:

```java
                if (!PacketAccessor.isParamsVersion()) {
                    PacketAccessor.PACK_OPTION.set(packet, 1);

                    if (PacketAccessor.VISIBILITY != null) {
                        PacketAccessor.VISIBILITY.set(packet, visible ? "always" : "never");
                    }
                } else {
                    // 1.17+
                    PacketAccessor.PACK_OPTION.set(packetParams, 1);

                    if (PacketAccessor.VISIBILITY != null) {
                        PacketAccessor.VISIBILITY.set(packetParams, visible ? "always" : "never");
                    }
                }
```

Reply:
oh lol, i didn't see it